CRM/Core/BAO/CustomField.php

```diff
@@ -725,7 +733,9 @@ public static function getFieldObject($fieldID) {
       if (!$field->find(TRUE)) {
         throw new CRM_Core_Exception('Cannot find Custom Field ' . $fieldID);
       }
-
+      // FIXME: This whole function should switch to APIv4 get which handles
+      // unserialization automatically.
+      $field->settings = CRM_Core_DAO::unSerializeField($field->settings, CRM_Core_DAO::SERIALIZE_JSON);
```

Comment:
this is generating `api_v3_ContactTest::testMergeGetConflicts
Failure in api call for Contact get_merge_conflicts:  Undefined property: CRM_Core_BAO_CustomField::$settings`

Reply:
Fixed.